frontend/src/app/types/app/complaints/outcomes/wildlife/drug-used.ts

```diff
@@ -35,3 +35,17 @@ export interface DrugUsedV2 {
 
   order: number;
 }
+
+export interface DrugUsedV3 {
```

Comment:
Same comment as above - probably don't need multiple versions here if the previous versions are dead code

Reply:
Cleaned up the types with b5d83d8.